src/components/layout/Layout.tsx

```diff
@@ -13,35 +13,47 @@ import { ABOUT_URL, APPLICANTS_URL, HOME_URL } from '../../constants';
 export const Layout: FC<ContainerProps> = ({ children, ...props }) => {
   const router = useRouter();
 
-  return (
-    <Container maxW='100%' p={0} {...props}>
-      <Box px={{ base: 5, md: 12 }} py={{ base: 3, md: 8 }}>
-        <Nav />
-      </Box>
-
-      {router.pathname === HOME_URL && (
+  function renderContent(): ReactNode {
```

Comment:
will change it to arrow functions since that is what we are using in most places

Reply:
done